style.css

```diff
@@ -57,82 +66,177 @@ body {
+  -webkit-align-items: center;
+  align-items: center;
 }
 
 .btn {
-	display: inline-block;
-	background-color: #D62B2E;
-	font-size: 32px;
-	text-align: center;
-	padding: 28px 85px;
-	border-radius: 10px;
+  display: inline-block;
+  background-color: #d62b2e;
+  font-size: 32px;
+  text-align: center;
+  padding: 28px 85px;
+  border-radius: 10px;
 }
 
 .btn:hover {
-	box-shadow: 3px 3px 8px #000;
-	cursor: pointer;
+  box-shadow: 3px 3px 8px #000;
+  cursor: pointer;
 }
 
 .cards_main {
-	padding-left: 140px;
-	background: url('images/card1.png')left no-repeat;
+  padding-left: 140px;
+  background: url('images/card1.png') left no-repeat;
 }
 
 .cards_main:hover {
-	background: url('images/card.png')left no-repeat;
+  background: url('images/card.png') left no-repeat;
 }
 
 .game_treecard {
   display: grid;
-  padding: 370px 360px;
   grid-template-columns: 1fr 1fr 1fr;
+  padding: 20px;
+  width: 730px;
+  position: absolute;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  top: 50%;
+  -webkit-transform: translate(0, -50%);
+  transform: translate(0, -50%);
 }
 
 .game_sixcard {
   display: grid;
-  padding: 160px 360px;
   grid-template-rows: 1fr 1fr;
   grid-template-columns: 1fr 1fr 1fr;
-  grid-row-gap: 364px;
+  grid-row-gap: 24px;
+  grid-column-gap: 24px;
+  padding: 20px;
+  width: 730px;
+  position: absolute;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  top: 50%;
+  -webkit-transform: translate(0, -50%);
+  transform: translate(0, -50%);
 }
 
 .game_tencard {
-	padding: 157px 126px;
+  padding: 157px 126px;
   display: grid;
   grid-template-rows: 1fr 1fr;
   grid-template-columns: repeat(5, 1fr);
-  grid-row-gap: 364px;
+  grid-row-gap: 24px;
+  grid-column-gap: 24px;
 }
 
 .card_box {
-  transition: transform 0.8s;
+  transition: transform .8s;
+  -webkit-transform-style: preserve-3d;
   transform-style: preserve-3d;
   display: inline;
-}
- 
-.shirt:hover {
-	cursor: pointer;
-	transform: translateY(-20px);
+  width: 218px;
+  height: 341px;
 }
 
 .shirt,
 .face {
   position: absolute;
-  width: 100%;
-  height: 100%;
+  -webkit-backface-visibility: hidden;
   backface-visibility: hidden;
 }
 
+.shirt:hover {
+  cursor: pointer;
+  -webkit-transform: translateY(-20px);
+  transform: translateY(-20px);
+}
 
 .face {
+  -ms-transform: rotateY(180deg);
+  -webkit-transform: rotateY(180deg);
+  -moz-transform: rotateY(180deg);
+  -o-transform: rotateY(180deg);
   transform: rotateY(180deg);
 }
 
 .flip {
+  -ms-transform: rotateY(180deg);
+  -webkit-transform: rotateY(180deg);
+  -moz-transform: rotateY(180deg);
+  -o-transform: rotateY(180deg);
   transform: rotateY(180deg);
 }
 
-
+@media (max-width: 920px) {
+  .columns {
+    flex-direction: column;
+  }
+}
+
+@media (max-width: 545px) {
+  .cards_main {
+    display: none;
+  }
+
+  .card_box,
+  .shirt,
+  .face {
+    width: 80px;
+    height: 150px;
+  }
+
+  .picture {
+    width: 100%;
+  }
+
+  .title {
+    font-size: 30px;
+    margin-bottom: 10px;
+}
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/evagame/pullRequest?prid=5788637)